src/main/java/ladder/domain/model/Points.java

```diff
@@ -0,0 +1,23 @@
+package ladder.domain.model;
+
+import ladder.domain.model.strategy.ConnectionStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Points {
+    private List<Boolean> points = new ArrayList<>();
+
+
+    public void connectLine(ConnectionStrategy connectionStrategy) {
+        if (connectionStrategy.isConnection()) {
+            points.add(true);
+            return;
+        }
+        points.add(false);
+    }
```

Comment:
다음과 같이 간단하게 표현도 가능할 것 같아요!!😆

```suggestion
    public void connectLine(ConnectionStrategy connectionStrategy) {
        points.add(connectionStrategy.isConnection());
    }
```